### PR TITLE
Bug/1110 compound metadata error handling

### PR DESCRIPTION
**Pull request by kzangeli**
Implements #1110 

Stupid bug fixed:

    return "r";

=>

    return r;

Plus a func-test to test the example of issue #1110

### CODE REVIEW

File: test/functionalTest/cases/1110_compound_metadata_error_handling/compound_metadata_error_handling.test [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
```
Comment (fgalan) [inline test/functionalTest/cases/1110_compound_metadata_error_handling/compound_metadata_error_handling.test]: I understasnd that the cases/1110_compound_metadata_error_handling will be removed once compounds for metadatas will be implemented. I'd suggest to add a FIXME to make that explicit.

Comment (kzangeli) [inline test/functionalTest/cases/1110_compound_metadata_error_handling/compound_metadata_error_handling.test]: sure.

Comment (kzangeli) [inline test/functionalTest/cases/1110_compound_metadata_error_handling/compound_metadata_error_handling.test]: Fixed in d45cf9d

**Comment by fgalan** [comment]
LGTM

**Pull request merged**